lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsFileSystemConfig.java

```diff
@@ -33,6 +33,12 @@
 
 public class GcsFileSystemConfig
 {
+    public enum AuthType
+    {
+        ACCESS_TOKEN,
+        DEFAULT;
```

Comment:
Is there an authentication scheme named "DEFAULT" for GCS? Or did you pick that value as a way to represent the default scheme to use when nothing else is chosen? If so, let's use the actual name of the authentication scheme.